README.md

```diff
@@ -3,13 +3,141 @@ This is a monorepo that contains Okta's OpenID Connect JavaScript resources.
+| [**jwt-verifier**](/packages/jwt-verifier)           | [![npm version](https://img.shields.io/npm/v/@okta/jwt-verifier.svg?style=flat-square)](https://www.npmjs.com/package/@okta/jwt-verifier)           | Easily verify JWTs from Okta                                                    |
+| [**okta-angular**](/packages/okta-angular)           | [![npm version](https://img.shields.io/npm/v/@okta/okta-angular.svg?style=flat-square)](https://www.npmjs.com/package/@okta/okta-angular)           | Angular support for Okta                                                        |
+| [**oidc-middleware**](/packages/oidc-middleware)     | [![npm version](https://img.shields.io/npm/v/@okta/oidc-middleware.svg?style=flat-square)](https://www.npmjs.com/package/@okta/oidc-middleware)     | Middleware to easily add OpenID Connect to the Node.js framework of your choice |
+| [**okta-react**](/packages/okta-react)               | [![npm version](https://img.shields.io/npm/v/@okta/okta-react.svg?style=flat-square)](https://www.npmjs.com/package/@okta/okta-react)               | React support for Okta                                                          |
+| [**okta-react-native**](/packages/okta-react-native) | [![npm version](https://img.shields.io/npm/v/@okta/okta-react-native.svg?style=flat-square)](https://www.npmjs.com/package/@okta/okta-react-native) | React Native support for Okta                                                   |
+| [**okta-vue**](/packages/okta-vue)                   | [![npm version](https://img.shields.io/npm/v/@okta/okta-vue.svg?style=flat-square)](https://www.npmjs.com/package/@okta/okta-vue)                   | Vue.js support for Okta                                                         |
+
+## Configuration Reference
+
+Each package is configured to look for environment variables based on the application type.
+
+```bash
+# Navigate into a specific package
+cd packages/${packageName}
+
+# Set the following environment variables
+#
+# ISSUER        - your authorization server
+# CLIENT_ID     - the client ID of your app
+# CLIENT_SECRET - the client secret of your app, required for the oidc-middleware package
+# USERNAME      - username of app user, required for tests
+# PASSWORD      - password of app user, required for tests
+export ISSUER=https://{yourOktaDomain}/oauth2/default
+...
+```
+
+## Testing
+
+Since the workspace contains libraries for Single-Page and Web Applications, you will need to have created a SPA and Web App in your Okta org.
+
+### Prerequisites
+
+#### Create a SPA
+
+1. Applications > Add Application
+2. Select SPA
+3. Add the following **login redirect URI**:
+    - `http://localhost:8080/implicit/callback`
+    - `http://localhost:3000/implicit/callback`
```

Comment:
Is there a reason we need 2 ports for the samples/tests? Can we refactor to use only `8080`, which is the default in the Developer Console?

Reply:
Yep. This is one of the few items that are going to be picked up in a follow-up PR (related to #297)

Reply:
Cool!